lemur/certificates/service.py

```diff
@@ -560,18 +560,35 @@ def query_common_name(common_name, args):
     :return:
     """
     owner = args.pop("owner")
-    if not owner:
-        owner = "%"
-
     # only not expired certificates
     current_time = arrow.utcnow()
 
-    result = (
-        Certificate.query.filter(Certificate.cn.ilike(common_name))
-        .filter(Certificate.owner.ilike(owner))
-        .filter(Certificate.not_after >= current_time.format("YYYY-MM-DD"))
-        .all()
-    )
+    if common_name == "%" and not owner:
```

Comment:
if "%" in common_name and not owner:

Done this way, you can change between using == and ilike.

Reply:
moving to `ilike` since `==` was helping significantly.